docs/system-admin-guide/backup/README.md

```diff
@@ -7,28 +7,46 @@ keywords: system backup
 ---
 # Backup
 
-Unless disabled via the [configuration](../../installation-and-operations/configuration/#backup-enabled)
-users can make backups of the OpenProject installation from within the administration area.
-They either need to be an administrator or have the global permission to do so.
+## Enterprise cloud
```

Comment:
I would suggest putting this section (lines 10 to 14) in the the enterprise cloud guide and link to this page after that. Something like "Please refer to the system admin guide here how to learn how to perform backups" or similar.